enterprise_subsidy/apps/content_metadata/api.py

```diff
@@ -19,6 +22,22 @@
     ProductSources.TWOU.value: CourseModes.EXECUTIVE_EDUCATION.value,
 }
 
+CACHE_NAMESPACE = 'content_metadata'
+
+
+def content_metadata_cache_key(enterprise_customer_uuid, content_key):
+    """
+    Returns a versioned cache key that includes the customer uuid and content_key.
+    """
+    return versioned_cache_key(enterprise_customer_uuid, content_key)
+
+
+def request_cache():
+    """
+    Helper that returns a namespaced RequestCache instance.
+    """
+    return RequestCache(namespace=CACHE_NAMESPACE)
```

Comment:
RequestCache is thread-local, so do you even need to put the code version in the cache?  Every new instance, let alone new deployment, will use a different cache, right?  Perhaps you meant to use TieredCache instead?

Reply:
We'll eventually used TieredCache, I wanted to get this versioned-cache key down on paper to establish a pattern that makes key-based invalidation the "go-to" in this repo.